### PR TITLE
[time] initial integration

### PR DESCRIPTION
**Pull request by 0x34d**
[time](https://github.com/facebook/time) library by meta.

usage: [opentelemetry-collector-contrib](https://github.com/open-telemetry/opentelemetry-collector-contrib), [Time-Appliance-Project](https://github.com/opencomputeproject/Time-Appliance-Project), [aperture](https://github.com/fluxninja/aperture) .

@abulimov does this look good?

### CODE REVIEW

File: projects/time/project.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+homepage: "https://opensource.fb.com"
+language: go
+primary_contact: ""
```
Comment (0x34d) [inline projects/time/project.yaml:3]: I don't have your email ID: @abulimov

Comment (leoleovich) [reply projects/time/project.yaml:3]: Feel free to add mine: leoleovich@meta.com

**Comment by 0x34d** [comment]
@Alan32Liu ping.

Upstream agreement  https://github.com/facebook/time/issues/303  https://github.com/google/oss-fuzz/pull/11095#discussion_r1368441495

**Comment by Alan32Liu** [comment]
> @Alan32Liu ping.

Thanks @0x34d. Asking the panel now.

**Comment by jonathanmetzman** [comment]
@0x34d not sure if we can give a reward for this since the fuzzers were written by the project. Still want to merge?

**Comment by jonathanmetzman** [comment]
> > @0x34d not sure if we can give a reward for this since the fuzzers were written by the project. Still want to merge?
> 
> Alright, you can project matter more.

Sorry, not sure I understand?

**Comment by 0x34d** [comment]
> > > @0x34d not sure if we can give a reward for this since the fuzzers were written by the project. Still want to merge?
> > 
> > 
> > Alright, you can project matter more.
> 
> Sorry, not sure I understand?

Alright, merge this.
I won't ask for a reward; I understand. :)

**Pull request merged**